### DISCUSSION TITLE
Requirements for inclusion

### DISCUSSION THREAD

**Discussion by Mitsjol** [osmlab/name-suggestion-index#9002] [Q&A] [answered]
When is a store chain notable enough to be included in the NSI? Does it need a certain amount of branches or are there any other requirements?

There are a lot of smaller retail and restaurant chains in the Netherlands that sometimes have <10 branches and I'm wondering if they're notable enough to be included.

Thanks in advance,

**Comment by bhousel** [comment, marked as answer]
The NSI collection script gathers names used at least 50 times from the OSM Planet, but we sometimes include smaller chains if they are notable.  Because we are using Wikidata IDs as the brand identifier, the threshold is basically "whatever Wikidata considers notable".
https://www.wikidata.org/wiki/Wikidata:Notability

So small chains can be fine, but there does need to be some "serious and publicly available sources" written about them, which can be used to justify that brand having its own Wikidata page.

**Reply by Cj-Malone** [reply]
And while the release requires Wikidata, NSI still has entries without it. If someone cares enough, they could add others, they just need to be aware that it won't make it into iD/Rapid.

**Reply by LaoshuBaby** [reply]
> And while the release requires Wikidata, NSI still has entries without it. If someone cares enough, they could add others, they just need to be aware that it won't make it into iD/Rapid.

As of now, if an NSI entry does not have wikidata, it will not be suggested in editors (such as iD/RapiD).

The current threshold can be seen at:
https://github.com/ideditor/nsi-collector/blob/a2a019ba6edf1b09e1be7ba35ccfad5d4c00145d/collect_osm.js#L33-L36
(This is not a mandatory standard, it is just a threshold for automatically discovering new brands from planet files. If it is really important, it can be completed manually.)
